include/mbedtls/x509.h

```diff
@@ -109,6 +109,28 @@
 /* \} name */
 /* \} addtogroup x509_module */
 
+/*
+ * X.509 v3 Subject Alternative Name types.
+ *      otherName                       [0]     OtherName,
+ *      rfc822Name                      [1]     IA5String,
+ *      dNSName                         [2]     IA5String,
+ *      x400Address                     [3]     ORAddress,
+ *      directoryName                   [4]     Name,
+ *      ediPartyName                    [5]     EDIPartyName,
+ *      uniformResourceIdentifier       [6]     IA5String,
+ *      iPAddress                       [7]     OCTET STRING,
+ *      registeredID                    [8]     OBJECT IDENTIFIER
```

Comment:
Minor: One might add here how this relates to the ASN.1 tags found in the SubjectAlternativeNames extension, which would allow users to add and contribute support for further name types.

Reply:
Perhaps, but I don't think it's relevant for this section. This comment is explaining the reason \ origin of choosing these values for the definitions.
I think a better location is in the SAN parsing function documentation, or a KB article

Reply:
Ok!

Reply:
created https://github.com/ARMmbed/mbedtls/issues/2646 to track this, but I don't think this should block the PR

Reply:
Thanks!